pom.xml

```diff
@@ -71,6 +71,7 @@
     <commons.text.version>1.10.0</commons.text.version>
     <commons.validator.version>1.7</commons.validator.version>
     <curator.version>5.5.0</curator.version>
+    <daffodil.version>3.8.0</daffodil.version>
```

Comment:
You should retry with Daffodil 3.10.0 which is the latest version. Should be drop-in replacement.

Reply:
Done!